nuxeo-opencmis-tests/src/test/resources/OSGI-INF/test-unfollow-cmis-spec.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
```

Comment:
Also add a non-test version of this file, loaded by Nuxeo, with the property to `false` and a `<description>` explaining what this is for (see other extension points contributing to ConfigurationService). Make the test file `test-unfollow-cmis-spec.xml` have a `<require>` on the base default config.